LkeServices/LkeServices.csproj

```diff
@@ -9,13 +9,19 @@
     <None Remove="Setthings\**" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="AutoMapper" Version="7.0.1" />
+    <PackageReference Include="jose-jwt" Version="2.4.0" />
     <PackageReference Include="Lykke.Payments.Client" Version="1.0.8" />
     <PackageReference Include="Lykke.Service.Balances.Client" Version="1.1.2" />
     <PackageReference Include="Lykke.Service.ClientAccount.Client" Version="1.3.0" />
     <PackageReference Include="Lykke.Service.Assets.Client" Version="4.0.1" />
-    <PackageReference Include="Lykke.Service.Session.Client" Version="1.20.1" />
+    <PackageReference Include="Lykke.Service.ClientAccountRecovery.Client" Version="1.0.17-beta" />

```

Comment:
Package would be updated after stable nuget release of Lykke.Service.ClientAccountRecovery.

Reply:
Updated